main.go

```diff
@@ -110,6 +110,20 @@ func main() {
 			DefaultText: "0, ie profiling disabled",
 			EnvVars:     []string{"BAZEL_REMOTE_PROFILE_PORT"},
 		},
+		&cli.DurationFlag{
+			Name:        "http_read_timeout",
+			Value:       0,
+			Usage:       "The HTTP read timeout for a client request in seconds. (applies to the frontend not the proxy backend)",
```

Comment:
How about this rewording:
`The HTTP write timeout for a server response in seconds (does not apply to the proxy backends or the profiling endpoint).`

Reply:
sounds good!